layouts/joomla/form/field/textarea.php

```diff
@@ -42,12 +45,23 @@
  * @var   array    $options         Options available for this field.
  * @var   array    $inputType       Options available for this field.
  * @var   string   $accept          File types that are accepted.
+ * @var   boolean  $charcounter     Does this field support a character counter?
  */
 
+ // Initialize some field attributes.
+if ($charcounter)
+{
+	// Load the js file
+	Factory::getDocument()->getWebAssetManager()->enableAsset('short-and-sweet');
+	// Set the css class to be used as the trigger
+	$charcounter = 'charcount';
+	// Set the text
+	$counterlabel = 'data-counter-label="' . Text::_('COM_CONFIG_METADESC_COUNTER') . '"';
+}
 $attributes = array(
 	$columns ?: '',
 	$rows ?: '',
-	!empty($class) ? 'class="form-control ' . $class . '"' : 'class="form-control"',
+	!empty($class) ? 'class="form-control ' . $class . $charcounter . '"' : 'class="form-control ' . $charcounter . '"',
```

Comment:
```suggestion
	!empty($class) ? 'class="form-control ' . $class . $charcounter . '"' : 'class="form-control' . $charcounter . '"',
```
If my above suggestion is accepted we don't need the space here anymore.

Reply:
Or does `$class` already have a space at the end? If so, then forget my suggestions.